src/servers/src/mysql/handler.rs

```diff
@@ -331,6 +417,29 @@ fn format_duration(duration: Duration) -> String {
     format!("{}:{}:{}", hours, minutes, seconds)
 }
 
+fn replace_params_with_values(
+    plan: &LogicalPlan,
+    param_types: HashMap<String, Option<ConcreteDataType>>,
+    params: Vec<ParamValue>,
+) -> Result<LogicalPlan> {
+    debug_assert_eq!(param_types.len(), params.len());
+
+    let mut values = Vec::with_capacity(params.len());
+
+    for i in 0..param_types.len() {
```

Comment:
use `param_types.iter().enumerate()` to get both value and index so that we can avoid using `unwrap`